client/src/components/Error/Error.js

```diff
@@ -2,25 +2,35 @@ import React from 'react';
 import { NavLink } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './Error.css'
+import astronaut from './Images/astronaut.svg'
+import four from './Images/4.svg';
+import zero from './Images/0.svg';
+import error from './Images/error.svg'
+import not_found from './Images/page-not-found.svg'
+import planet from './Images/planet.svg'
 
 const Error = () => {
     return (
-        <div className="error-component container">
-            <hr />
-            <div className="text-center text-danger firstdiv">
-                <h1 className="font-weight-bolder" style={{ opacity: 1 }}>404 Error </h1>
-                <h4 className="font-weight-bolder">Page Not Found</h4>
-            </div>
-            <hr />
-            <div className="text-center seconddiv">
-                <div className="pad-topbot">
-                    <p className="whitecolor display-5">We're sorry , the page you requested could not be found</p>
-                    <p className="whitecolor">Please go back to the home page</p>
-                    <div className="text-center">
-                        <button type="button" className="btn btn-danger"><NavLink to="/" className=".text-muted">Back to home</NavLink></button>
-                    </div>
+        <div className="container-fluid mt-5 mx-auto">
+            <div className="stars">
+                <NavLink className="btn-go-home" to="/" >GO BACK</NavLink>
```

Comment:
I would suggest that in place of  `GO BACK`  here , it should be `Home` or `Back to Home` . I think it would be more accurate here according to its purpose/link.

Reply:
okk I will resolve this in my next commit .thanks  for suggestion!!